### ISSUE TITLE
Installation fails: "curl -L get.zplug.sh | zsh" results in "Could not resolve host: get.zplug.sh"

### ISSUE THREAD

**Issue by alexbarton** [zplug/zplug.github.io#3]
The zplug.github.io / zplug.sh site states:

> Installation
> curl -L get.zplug.sh | zsh

But this fails with "_Could not resolve host: get.zplug.sh_", because "get.zplug.sh" is not known in DNS:

```
~ $ dig +short -t A zplug.sh
104.31.77.99
104.31.76.99
~ $ dig +short -t A get.zplug.sh
~ $ dig +short -t AAAA get.zplug.sh
~ $
```

**Comment by ghost** [comment]
The way of the installation has changed.
`curl -sL zplug.sh/installer | zsh`

[Click here for details](https://github.com/zplug/zplug#installation)

**Comment by NigoroJr** [comment]
Oh, forgot that this was fixed with ce23b87fc72d70458a138d9a4b0b3b40be55d26f. Thanks for reporting!
